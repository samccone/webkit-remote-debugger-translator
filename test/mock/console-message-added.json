{
    "method": "Console.messageAdded",
    "params": {
        "message": {
            "column": 18,
            "level": "log",
            "line": 10,
            "parameters": [
                {
                    "type": "string",
                    "value": "hi"
                }
            ],
            "repeatCount": 3,
            "source": "console-api",
            "stackTrace": [
                {
                    "columnNumber": 18,
                    "functionName": "",
                    "lineNumber": 10,
                    "url": "http://samccone.github.io/webkit-remote-debugger-translator/test/sample-page/"
                }
            ],
            "text": "hi",
            "type": "log",
            "url": "http://samccone.github.io/webkit-remote-debugger-translator/test/sample-page/"
        }
    }
}
