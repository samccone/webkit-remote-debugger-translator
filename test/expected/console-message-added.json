{
    "method": "Console.messageAdded",
    "params": {
        "message": {
            "column": 15,
            "executionContextId": 3,
            "level": "log",
            "line": 10,
            "parameters": [
                {
                    "type": "string",
                    "value": "hi"
                }
            ],
            "source": "console-api",
            "stackTrace": [
                {
                    "columnNumber": 15,
                    "functionName": "",
                    "lineNumber": 10,
                    "scriptId": "60",
                    "url": "http://samccone.github.io/webkit-remote-debugger-translator/test/sample-page/"
                }
            ],
            "text": "hi",
            "timestamp": 1446248362.35488,
            "type": "log",
            "url": "http://samccone.github.io/webkit-remote-debugger-translator/test/sample-page/"
        }
    }
}
